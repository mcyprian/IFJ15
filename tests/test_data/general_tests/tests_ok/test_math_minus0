int main()
{
    double a = 3.4 - 2.2;
    cout << a << 3.14 << "\n";

    if("q" != "d")
    {
        cout << "ok\n";
    }
    else
    {}

    if("b" >= "a")
    {
        cout << "ok\n";
    }
    else
    {}
    
    if("a" >= "a")
    {
        cout << "ok\n";
    }
    else
    {}
    
    if("a" <= "b")
    {
        cout << "ok\n";
    }
    else
    {}
    
    if("b" <= "b")
    {
        cout << "ok\n";
    }
    else
    {}

    double x = "a" >= "c";
    int f = "a" <= "f";

    cout << x << f << "\n";

    return 42;

}
