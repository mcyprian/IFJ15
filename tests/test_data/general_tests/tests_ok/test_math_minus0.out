1.23.14
ok
ok
ok
ok
ok
01
